src/bindings/Next.res

```diff
@@ -57,11 +57,11 @@ module Link = {
     ~href: string,
     ~_as: string=?,
     ~prefetch: bool=?,
-    ~replace: option<bool>=?,
```

Comment:
Is this how it works in ReScript 11 now due to uncurried-by-default? Or was this wrong ever since?

Reply:
JSX v4 mode https://rescript-lang.org/docs/react/latest/migrate-react#bindings-to-js-components-with-optional-props